.maestro/flows/production.yaml

```
@@ -0,0 +1,12 @@
+appId: app.exactly
+tags:
+  - production
+---
+- launchApp: { clearState: true, permissions: { all: deny, camera: allow } }
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)